nginx.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nginx:1.19.6-alpine
```

Comment:
Why not `nginx:mainline-alpine instead`? Or even `nginx:stable-alpine`?

Pinning the version requires maintenance in the future, and this isn't data-sensitive either for it to require so.

Reply:
@3nprob Can you check this out?

Reply:
Reproducability mainly - but agreed, changed to `nginx:mainline-alpine` now